torch/nn/modules/activation.py

```diff
@@ -741,19 +744,20 @@ def _reset_parameters(self):
     @weak_script_method
     def forward(self, query, key, value, key_padding_mask=None, incremental_state=None,
                 need_weights=True, static_kv=False, attn_mask=None):
-        """
-        Inputs of forward function
-            query: [target length, batch size, embed dim]
-            key: [sequence length, batch size, embed dim]
-            value: [sequence length, batch size, embed dim]
-            key_padding_mask: if True, mask padding based on batch size
-            incremental_state: if provided, previous time steps are cashed
-            need_weights: output attn_output_weights
-            static_kv: key and value are static
-
-        Outputs of forward function
-            attn_output: [target length, batch size, embed dim]
-            attn_output_weights: [batch size, target length, sequence length]
+        r"""
+    Inputs
+        query: [target length, batch size, embed dim].
+        key: [sequence length, batch size, embed dim].
+        value: [sequence length, batch size, embed dim].
+        key_padding_mask: if True, mask padding based on batch size.
+        incremental_state: if provided, previous time steps are cashed.
+        need_weights: output attn_output_weights.
+        static_kv: key and value are static.
+        attn_mask: mask to avoid attn to learn from certain positions.
```

Comment:
```suggestion
        attn_mask: mask that prevents attention from attending to certain positions.
```
Also the size and type of this object should be added in using the standard parenthesis based format.